notebooks/02_Fairing/02_01_fairing_introduction.ipynb

```diff
@@ -176,6 +168,16 @@
     "!eval $(aws ecr get-login --no-include-email --region=$REGION)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create an ECR repository in the same region\n",
```

Comment:
What if people already have repository? it throws errors or something different? This repo is not just for workshop. Can we add one more step to check repo first and then ask you to create if it doesn't exist?

Reply:
Added comment on this. Resolving